src/EditorFeatures/CSharpTest/SplitStringLiteral/SplitStringLiteralCommandHandlerTests.cs

```diff
@@ -23,11 +23,11 @@ public class SplitStringLiteralCommandHandlerTests
         {
             using (var workspace = TestWorkspace.CreateCSharp(inputMarkup))
             {
-                var document = workspace.Documents.Single();
+                Assert.True(workspace.TryGetDocumentWithSelectedSpan(out var doc, out var originalSelection));
+                var document = workspace.Documents.Single(d => d.Id == doc.Id);
```

Comment:
What's the difference between `document` and `doc`?

Reply:
TestWorkspace have two different document concepts.  One is the workspace doc (teh doc you know and love), the other is the test 'doc'.  this has the actual test state on it, like our own ITextBuffer/ITextView and whatnot since we're not actually in VS.